portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Renaud's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <script src="https://kit.fontawesome.com/6094c53d83.js" crossorigin="anonymous"></script>
+  </head>
+  <body class="body">
+    <nav>
+       <div class="container">
+        <h1> Fred </h1>
+
+        <div class="menu">
+            <a href="index.html" class="is-active">Home</a>
+            <a href="about_me.html">About Me</a>
+            <!--<a href="gallery.html">Gallery</a>-->
+            <a href="contact.html">Conatact</a>
+        </div>
+
+        <buttom class="ham">
+            <span></span>
+            <span></span>
+            <span></span>
+        </buttom>
+
+       </div>
+    </nav>
+
+    <div id="content">
+      <div id="main">
+    
+      <h4>Hi here you will find my contact info</h4>
+
+      
+      <h4>Here are my social medias</h4>
+      <a href="https://www.linkedin.com/in/renaud-fred-noubieptie-kamgang-87b839188/"> <i class="fa-brands fa-linkedin fa-6x"></i></a>
```

Comment:
For links, you can put it like this: \<a href="YOUR_LINK">[the text for the link]\</a>.
e.g: \<a href="link_to_my_linkedin">my linkedin\</a>, this acts like the hyperlink

Reply:
Actually the code in between the link provides the icon. For that specific one its the linkedin icon.